docs/examples/affinity/cookie/README.md

```diff
@@ -50,13 +54,13 @@ Date: Fri, 10 Feb 2017 14:11:12 GMT
 Content-Type: text/html
 Content-Length: 612
 Connection: keep-alive
-Set-Cookie: INGRESSCOOKIE=a9907b79b248140b56bb13723f72b67697baac3d; Path=/; HttpOnly
+Set-Cookie: INGRESSCOOKIE=a9907b79b248140b56bb13723f72b67697baac3d; Expires=Wed, 07-Nov-18 09:05:06 GMT; Max-Age=172800; Path=/; HttpOnly
```

Comment:
Are you sure this is the correct expiry given the annotation is set to  172800? Isn’t the annotation value a Unix timestamp ?

Reply:
Sure, there is a screenshot from my test enviroment
<img width="1315" alt="cookie-expires" src="https://user-images.githubusercontent.com/8757567/48040536-50c12c00-e1b4-11e8-9635-e7095780d038.png">

Reply:
@ElvinEfendi Any more comments?

Reply:
@skeeey as you can see from your screenshot it is set to `03 January 1970` which is expected given the way you implemented the feature. However in the docs you say it will be `07 Nov 2018`.

Reply:
`01 January 1970` is Unix epoch time, same in every Unix based system.